app/helpers/application_helper.rb

```diff
@@ -382,7 +382,7 @@ def topic_featured_link_domain(link)
 
   def theme_ids
     if customization_disabled?
-      nil
+      [nil]
```

Comment:
I believe we've discussed this before but an empty array with a single `nil` value is really weird to me. (I believe last time you said it is required by some existing code.)

You should make a note that this is in need of a refactor to a more sensible structure. If there's a reason why an empty array or nil wouldn't work, maybe the answer is creating a helper class that can establish the difference with some other properties.

Reply:
Yes, this is because of safe mode - the array needs to have at least one element, since it gets looped over to generate the CSS. No elements = absolutely no CSS 😢 

I'll look into refactoring this